Cargo.toml

```diff
@@ -9,11 +9,18 @@ default= []
 services=["tonic-build", "tonic"]
 
 [dependencies]
-tonic = {version = "0", optional = true}
+base64 = ">=0.21"
+byteorder = "1"
 bytes = "1"
 prost = "0"
+rand = "0.8"
+rand_chacha = ">=0.3"
+sha2 = "0"
+rust_decimal = { version =  "1", features = ["serde-with-float"] }
 serde = {version = "1", features = ["derive"]}
 serde_json = "1"
+tonic = {version = "0", optional = true}
+thiserror = "1.0"
```

Comment:
Do we need to pin this less strict at `"1"`?

Reply:
check [a45d99](https://github.com/helium/proto/pull/334/commits/9a45d99267e757def7782d339f7c63e78d2193bd)